sim_data/plotter.py

```diff
@@ -35,8 +35,13 @@
 rocketY = []
 rocketZ = []
 
-data_lists = [timestamps, rx, ry, rz, vx, vy, vz, ax, ay, az, fx, fy, fz,
-              q0, q1, q2, q3, roll, pitch, yaw, rocketX, rocketY, rocketZ]
+sensorX = []
+sensorY = []
+sensorZ = []
+
```

Comment:
Are sensorX, sensorY and sensorZ the measurements of the sensors? Add comments here, slightly unclear

Reply:
Yeah these arrays are filled with measurements from the simulated accelerometer right now. Again, not trying to make the python side pretty right now, just trying to see if the sensors work.